Signal/src/ViewControllers/MediaDetailViewController.m

```diff
@@ -44,6 +44,9 @@ - (BOOL)canPerformAction:(SEL)action withSender:(nullable id)sender
 
 #pragma mark -
 
+// This approximates the curve of our message bubbles, which makes the animation feel a little smoother.
+const CGFloat MediaDetailViewControllerMediaCornerRadius = 17;
```

Comment:
Can you please pull out a constant in `OWSMessageCell` and use it here?
```
        maskedSubview.layer.cornerRadius = 17;
```

Reply:
Yes, done.